src/styles/blocks/content.scss

```diff
@@ -0,0 +1,21 @@
+.content {
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  padding: 50px $container-paddings;
+  max-width: $container-max-width;
+  margin: 0 auto;
```

Comment:
You can't use margins on BEM blocks, use mix of element and block for that purpose